web/src/components/FormActivityApproach.js

```diff
@@ -11,14 +11,16 @@ const Approaches = ({ fields, meta: { error, submitFailed } }) => (
     {fields.map((approach, idx) => (
       <div key={approach} className="mb3 sm-col-8">
         <div className="relative">
-          <button
-            type="button"
-            className="absolute right-0"
-            title="Remove Goal"
-            onClick={() => fields.remove(idx)}
-          >
-            Remove
-          </button>
+          {idx !== 0 && (
```

Comment:
Wondering if this should be something like `fields.length !== 0`?

Reply:
the intention here is that you should not be allowed to delete the approach input if it's the only one on the page, but if you add multiple approaches, you can delete those.

Reply:
if we use `fields.length`, then i think it will either allows have a delete button or never have one (for each input), but i think we want something slightly different (all but the first having the delete button)

Reply:
What if the first one is the only one they want to delete?  That was the situation I was thinking of.  Like, I have five fields, and the 2nd-5th ones are right, but I want to get rid of that first one.

Reply:
ahh, good point, will tweak 👍 🙌